src/main/java/org/jabref/logic/bibtexkeypattern/BracketedPattern.java

```diff
@@ -41,9 +43,54 @@
 public class BracketedPattern {
     private static final Logger LOGGER = LoggerFactory.getLogger(BracketedPattern.class);
 
-    private static final String STARTING_CAPITAL_PATTERN = "[^A-Z]";
     private static final int CHARS_OF_FIRST = 5;
-    private static final Pattern REGEX_PATTERN = Pattern.compile(".*\\(\\{([A-Z]+)\\}\\).*");
+
+    /** Matches everything that is not an uppercase ASCII letter */
+    private static final Pattern NOT_CAPITAL_FIRST_CHARACTER = Pattern.compile("[^A-Z]");
+    /** Matches with "({[A-Z]}+)", which should be used to abbreviate the name of an institution */
+    private static final Pattern ABBREVIATIONS = Pattern.compile(".*\\(\\{[A-Z]+}\\).*");
+    /** Matches with "dep"/"dip", case insensitive */
+    private static final Pattern DEPARTMENTS = Pattern.compile("^d[ei]p.*", Pattern.CASE_INSENSITIVE);
+    private enum INSTITUTION {
```

Comment:
Enum should have normal capitalization, i.e. `Institution`. The same holds true for the enum values

Reply:
@tobiasdiez  No! In java Enum are all Uppercase see here:
>Because they are constants, the names of an enum type's fields are in uppercase letters.
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Good catch and thanks for the link! Updating INSTITUTION -> Institution and I'll try to add it to my checkstyle.